cfme/tests/candu/test_host_graph.py

```diff
@@ -107,9 +107,9 @@ def refresh():
     assert graph_data > 0
 
 
-@pytest.mark.uncollectif(lambda provider, interval:
+@pytest.mark.uncollectif(lambda provider, graph_type, interval:
```

Comment:
Required:
As you are removing `interval` check please remove this as well. Please add run for test.

Reply:
Done